sorc/build_regrid_nemsio.sh

```diff
@@ -18,12 +18,13 @@ export F90=${FCMP}
 export LD=${FCMP}
 export F77=${FCMP}
 
+export NETCDF_LDFLAGS="-L${NETCDF}/lib -lnetcdff -lnetcdf -L${HDF5_LIBRARIES} -lhdf5_hl -lhdf5 -lz"
 export FCFFLAGS="" # "-convert native -assume byterecl -heap-arrays -mcmodel=large -shared-intel"
 export LDFLAGS="${FCFFLAGS}"
 export OPTIMIZATION="-O3" #-axCORE-AVX2,AVX -xSSE4.2 -O3
 export DEBUG="-traceback -g" #-O0 #-C #-fp-stack-check #-check all -fp-stack-check
 
-if [ $target != hera ]; then
+if [ $target = wcoss2 ]; then
```

Comment:
If we're going to use this, why aren't we just using it on every machine?

Reply:
Okay, leave it like this for now. I don't know much about this tool, so probably something for @aerorahul to look at and see if there is something the stack team needs to fix.

Reply:
Note, it also doesn't work on Hera, same error. Here is the compilation command when I try:
```
mpiifort  kinds.o constants.o physcons.o mpi_interface.o namelist_def.o variable_interface.o netcdfio_interface.o interpolation_interface.o gfs_nems_interface.o fv3_interface.o  -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack//intel-18.0.5.274/impi-2018.0.4/netcdf/4.7.4/include -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack//intel-18.0.5.274/impi-2018.0.4/netcdf/4.7.4/include -I/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/impi-2018.0.4/nemsio/2.5.2/include main.f90 /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/impi-2018.0.4/nemsio/2.5.2/lib/libnemsio.a /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/bacio/2.4.1/lib/libbacio_4.a /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/w3nco/2.4.1/lib/libw3nco_d.a /scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/intel-18.0.5.274/sp/2.3.3/lib/libsp_4.a -L/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack//intel-18.0.5.274/impi-2018.0.4/netcdf/4.7.4/lib -lnetcdff -o regrid_nemsio > log.make.regrid_nemsio
```